spec/resources/schema_dot_org/test.html

```diff
@@ -1,4 +1,4 @@
-<html>    
+<!--<html>    
```

Comment:
Name this file something more explicit, like 'html_with_schema_dot_org_but_missing_required_elements.html'